precode.sh

```diff
@@ -6,38 +6,60 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p ./task ./task/dir1 ./task/dir2 ./task/dir3 ./task/dir3/dir4
```

Comment:
Можно использовать флаг `-p`, тогда не нужно указывать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```